web/scripts/generateSitemap.js

```diff
@@ -0,0 +1,32 @@
+// Generates a static sitemap for the app based on the zones in the config file
+
+import fs from 'node:fs';
+import path from 'node:path';
+
+import zonesConfig from '../config/zones.json' assert { type: 'json' };
```

Comment:
Is our node version during the earthly build process pegged to a specific version or will it default to the most recent major LTS? I think Node updated [import asserts -> import attributes](https://nodejs.org/docs/latest-v20.x/api/esm.html#import-attributes) in >20.10.0 and not I'm not certain if we need to handle that.

Reply:
Otherwise, it looks good! Really love that we won't have to manually manage it :D

Reply:
Ah good spotting, it should use the latest 20.x version so should be fine to switch. I'll go ahead and switch, if it passes CI it should be fine.

Reply:
So slight problem, prettier screams at me for using `with`. We'll have to keep this for now and then switch later once we have upgraded prettier (which is not as straight forward as it sounds sadly).

Reply:
Prettier whyy 😭 - SGTM

Reply:
Pretty sure this one is on us, we use Prettier 2 so we can do some hacky thing to avoid creating a top level package.json config. Sounds like we might have to bite the bullet and do it though. Probably as a workspace so we don't duplicate dependencies.